niaid.bionimbus.org/portal/gitops.json

```diff
@@ -245,6 +245,21 @@
       "dataType": "subject",
       "nodeCountTitle": "Subjects",
       "fieldMapping": [
+        { "field": "frstdhd", "name": "Year of death" },
+        { "field": "arthxbase", "name": "ART Use Prior to Baseline"},
+        { "field": "bshbvstat", "name": "Baseline HBV Sero-status"},
+        { "field": "bshcvstat", "name": "Baseline HCV Sero-status"},
+        { "field": "cd4nadir", "name": "Year of Death"},
```

Comment:
does `cd4nadir` and `frstdhd` all means `Year of death`?

Reply:
Whoops. No. How can I change?

Reply:
you can change in your local branch and push to this remote branch again. But remember to `git pull` first on your local branch ;)